### PR TITLE
feat(quiz): add counter for number of questions in a quiz round

### PR DESCRIPTION
**Pull request by jesus-s-sanzo**
Numbers next to the questions in the quiz overview to help to identify them.
image

### CODE REVIEW

File: hosting/src/app/games/quiz/lobby/lobby.component.html [hunk 1]
```diff
@@ -65,8 +65,8 @@ <h2>Quiz Overview</h2>
                 <div class="round-list__item">
                     <brand-expansion label="{{ round.title }} ({{ round.questionList | count }})" (onToggle)="loadQuestions(round)">
                         <ng-container *ngIf="round.questions; else noQuestions">
-                            <div class="question" *ngFor="let question of round.questions">
-                                <div>{{ question.text | slice: 0 : 30 }}</div>
+                            <div class="question" *ngFor="let question of round.questions; index as i">
+                                <div><span class="qNumber">{{i+1}}</span> {{question.text | slice: 0 : 29 }}</div>
```
Comment (eddiejaoude) [inline hosting/src/app/games/quiz/lobby/lobby.component.html]: Class name to be hyphenated and spaces around braces ...

```suggestion
                                <div><span class="question-number">{{ i + 1 }}</span> {{ question.text | slice: 0 : 29 }}</div>
```

File: hosting/src/app/games/quiz/lobby/lobby.component.scss [hunk 2]
```diff
@@ -15,6 +15,9 @@
         margin: 0.25em 0;
         padding-left: 0.25em;
         border-left: 4px solid $brand-primary;
+        .qNumber {
+            color: $brand-primary;
+        }
```
Comment (eddiejaoude) [inline hosting/src/app/games/quiz/lobby/lobby.component.scss]:
```suggestion
        border-left: 4px solid $brand-primary;

        .question-number {
            color: $brand-primary;
        }
```

Comment (eddiejaoude) [reply hosting/src/app/games/quiz/lobby/lobby.component.scss]: Could also use css increments https://www.w3schools.com/cssref/pr_gen_counter-increment.asp

**Comment by jesus-s-sanzo** [comment]
Counter was re-implemented using CSS counter-increment
css class renamed to kebab-case

**Comment by eddiejaoude** [comment]
Looks good 👍 - I will test it out later on. Thank you 🤓

File: hosting/src/app/games/quiz/lobby/lobby.component.html [hunk 3]
```diff
@@ -70,7 +70,7 @@ <h2>Quiz Overview</h2>
                     <brand-expansion label="{{ round.title }} ({{ round.questionList | count }})" (onToggle)="loadQuestions(round)">
                         <ng-container *ngIf="round.questions; else noQuestions">
                             <div class="question" *ngFor="let question of round.questions">
-                                <div>{{ question.text | slice: 0 : 30 }}</div>
+                                <div><span class="question-number"></span> {{ question.text | slice: 0 : 29 }}</div>
```
Comment (andrew-cunliffe) [inline hosting/src/app/games/quiz/lobby/lobby.component.html]: the span to provide a class isn't really needed, you can just use the div in the css selector

```
.question {
    > div {
        &::before {
            ......
        }
    }
}
```

keeps the HTML a bit cleaner (and I don't like empty html tags if they can be avoided :) )

Comment (jesus-s-sanzo) [reply hosting/src/app/games/quiz/lobby/lobby.component.html]: Done.
We could even get rid of the inner div by adding adding display:flex to the question class like this:
` .question {
    ...
        display:flex;
        &::before {
            color: $brand-primary;
            ...
        }
    }`

I guess you put it there to give it further use in the future so I kept it.

**Pull request merged**